apps/charterafrica/src/utils/translationConstants.js

```diff
@@ -5,7 +5,8 @@ const labelsPerLocale = {
     "-topic": "Topic Z-A",
     views: "Views Ascending",
     "-views": "Views Descending",
-    stars: "Stars Ascending",
+    stars: "Stars",
+    starsAscending: "Stars Ascending",
```

Comment:
```suggestion
    "+stars": "Stars Ascending",
```

This should go well with `"-stars"` for descending.